crux-bench/src/crux/bench/watdiv.clj

```diff
@@ -0,0 +1,483 @@
+(ns crux.bench.watdiv
+  (:require [clojure.java.io :as io]
+            [clojure.tools.logging :as log]
+            [crux.bench :as bench]
+            [crux.api :as crux]
+            [crux.index :as idx]
+            [crux.rdf :as rdf]
+            [crux.sparql :as sparql]
+            [datomic.api :as d]
+            [crux.io :as cio]
+            [clojure.string :as string])
+  (:import [java.io Closeable File StringReader]
+           java.util.Date
+           org.eclipse.rdf4j.query.Binding
+           org.eclipse.rdf4j.repository.RepositoryConnection
+           org.eclipse.rdf4j.repository.sail.SailRepository
+           org.eclipse.rdf4j.rio.RDFFormat
+           org.eclipse.rdf4j.sail.nativerdf.NativeStore))
+
+(defn output-to-file [out output]
+  (spit out (str output "\n") :append true))
+
+(def watdiv-tests
+  {"watdiv-stress-100/warmup.1.desc" "watdiv-stress-100/warmup.sparql"
+   "watdiv-stress-100/test.1.desc" "watdiv-stress-100/test.1.sparql"
+   "watdiv-stress-100/test.2.desc" "watdiv-stress-100/test.2.sparql"
+   "watdiv-stress-100/test.3.desc" "watdiv-stress-100/test.3.sparql"
+   "watdiv-stress-100/test.4.desc" "watdiv-stress-100/test.4.sparql"})
+
+(def query-timeout-ms 30000)
+
+(def watdiv-input-file (io/resource "watdiv.10M.nt"))
+
+(defn execute-stress-test
+  [conn query-fn test-name num-tests ^Long num-threads]
+  (bench/run-bench (keyword test-name)
```

Comment:
`run-bench` has `{:style/indent 1}` metadata on it to suggest to editors that it has one 'special' arg, and everything else is 2-space indented (because it's a block) - I'm guessing this is vim? :smile: